contracts/snowglobes/traderJoe/snowglobe-joe-avax-fief.sol

```diff
@@ -0,0 +1,136 @@
+// https://github.com/iearn-finance/vaults/blob/master/contracts/vaults/yVault.sol
+
+pragma solidity ^0.6.7;
+
+import "../../interfaces/controller.sol";
+
+import "../../lib/erc20.sol";
+import "../../lib/safe-math.sol";
+
+contract SnowGlobeJoeAvaxFief is ERC20 {
+    using SafeERC20 for IERC20;
+    using Address for address;
+    using SafeMath for uint256;
+
+    IERC20 public token;
+
+    uint256 public min = 9500;
+    uint256 public constant max = 10000;
+
+    address public governance;
+    address public timelock;
+    address public controller;
+
+    constructor(
+        address _token,
+        address _governance,
+        address _timelock,
+        address _controller
+    )
+        public
+        ERC20(
+            string(abi.encodePacked("freezing ", ERC20(_token).name())),
+            string(abi.encodePacked("s", ERC20(_token).symbol()))
+        )
+    {
+        _setupDecimals(ERC20(_token).decimals());
+        token = IERC20(_token);
+        governance = _governance;
+        timelock = _timelock;
+        controller = _controller;
+    }
+
+    function balance() public view returns (uint256) {
+        return
+            token.balanceOf(address(this)).add(
+                IController(controller).balanceOf(address(token))
+            );
+    }
+
+    function setMin(uint256 _min) external {
+        require(msg.sender == governance, "!governance");
+        require(_min <= max, "numerator cannot be greater than denominator");
+        min = _min;
+    }
+
+    function setGovernance(address _governance) public {
+        require(msg.sender == governance, "!governance");
+        governance = _governance;
+    }
+
+    function setTimelock(address _timelock) public {
+        require(msg.sender == timelock, "!timelock");
+        timelock = _timelock;
+    }
+
+    function setController(address _controller) public {
+        require(msg.sender == timelock, "!timelock");
+        controller = _controller;
+    }
+
+    // Custom logic in here for how much the globes allows to be borrowed
```

Comment:
Let's get our comments in line with the [natspec format](https://docs.soliditylang.org/en/v0.8.11/natspec-format.html).

Also note the difference in the intended audience of dev & notice.

Here's a decent example from our codebase:
https://github.com/Snowball-Finance/axial-staking/blob/main/contracts/StakingVe.sol

Reply:
I'm still not 100% clear on when `@dev` should be used. It seems `@notice` is for description of the function and `@dev` is extra sauce in the form of implementation details?

Reply:
In the natspec documentation, it demonstrates how the dev tag is used.  It should be used in a natspec function header to communicate any technical information which may be useful to developers.  These could be developers forking your code in the future, developers interacting with your code via other contracts, etc.

If you look at the example bayological linked above, you will notice that the dev header is used to point out when our web application might call something.